game.js

```diff
@@ -0,0 +1,94 @@
+
+var userName = prompt('What\'s your name?'); //This is asking the user what their name is
+console.log('The visitor\'s name is: ' + userName); //This is logging the visitors answer to the system
+var rightAnswer = 0;
+var wrongAnswer = 0;
+
+var favFood = prompt('Do I like tacos?'); //This is a prompt that pops up and asks if I like tacos.
+if (favFood.toLowerCase() === 'yes' || 'y' || 'ye') {
+  alert('You are correct, ' + userName + '. I love tacos.'); // This will alert the visitor if they typed yes into the entry box.
+  rightAnswer++; //If the user gets the right answer they get 1 point.
+  console.log('This answer was correct. ' + userName + ' gets a point.');
+  console.log('The user entered:' + favFood + ' which equates to Yes.'); //This is logging the visitors answer to the system
+} else {
+    alert('You are incorrect, ' + userName + '. I love Tacos.'); // This is what the viewer will see if they answer anything besides yes
```

Comment:
Try adding a conditional with your else's. When I typed in 'no' for your answers, it alerted me saying it was correct. This also applies to your 2, 3, and 4th question. Let me know if you have any questions.